src/services/transaction-service.js

```diff
@@ -0,0 +1,11 @@
+import { StorageService } from "./storage";
+
+export class TransactionListService {
+  get() {
+    return StorageService.get("transactionList");
+  }
+
+  set(value) {
```

Comment:
get and set should be static methods -- so you won't need to instantiate this class